tests/lib/installers.py

```diff
@@ -358,6 +358,22 @@ def install_IRCserver(device):
         device.expect(device.prompt)
     except:
         device.expect(device.prompt)
+        device.sendline('apt-get install update-inetd -y') # Update inetd before installation
+        device.expect(device.prompt, timeout=90)
```

Comment:
You meant to include this as well? These make sense as two separate commits or at least add the fact you are fixing an issue with IRC installation as well in the commit message.

Reply:
Update the commit message Matt . Hope this makes sense now.

Reply:
No the commit does not make sense, it says nothing about why we are updating the IRC function.

Also if you could make the lines 80 chars or less....

Reply:
Sorry for the Mess Matt. I have address your comments and combined the 2 changes and committed.
https://github.com/lgirdk/boardfarm/commit/185b20dfd7d20827af346905fe5116ee24c4cd0a